src/utils/defaults.js

```diff
@@ -1,18 +1,13 @@
-import { isFile, readFileSync } from './fs.js';
+import { isFile, readFile } from './fs.js';
 import { dirname, isAbsolute, resolve } from './path.js';
 import { blank } from './object.js';
 
-export function load ( id ) {
-	return readFileSync( id, 'utf-8' );
```

Comment:
You still should specify 'utf-8'. Just replace with promise version, not delete.